File: lecture_004/array/get_display.cpp
#include<iostream>
#include<vector>
using namespace std;

    void get(vector<int>&arr,int n) {
        cout<<"Enter data..";
        for(int i=0;i<n;i++){
            cin>>arr[i];
        }
    }       
    void display(vector<int>&arr,int n) {
        cout<<"Inserted array is..";
        for(int i=0;i<n;i++){
            cout<<arr[i]<<" ";
        }
    }


int main(int args, char **argv){
    int n;
    cout<<"Enter size of array..";
    cin>>n;
    vector<int>arr(n,0);
    get(arr,n);
    display(arr,n);
    return 0;
}

File: lecture_004/array/get_display.java
import java.util.Scanner;
public class get_display{
    public static Scanner scn=new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Enter size of array");
        int n=scn.nextInt();
        int[]arr=new int[n];
        get(arr);
        display(arr);
    }
    public static void get(int[] arr) {
        System.out.println("Enter data..");
        for(int i=0;i<arr.length;i++){
            arr[i]=scn.nextInt();
        }
    }       
    public static void display(int[] arr) {
        System.out.println("Inserted array is..");
        for(int i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
    }
}